capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -80,12 +81,30 @@ export class SearchUserModal extends Component {
     // TODO: Update ClubPage based on user input
 
     if (this.props.club && this.props.type === 'clubs' && this.props.club.memberIDs) {
-      if (this.props.club.memberIDs.length > 0) {
-        this.fetchMembers();
-      }
+        if (this.props.club.memberIDs.length > 0) {
+          this.fetchMembers();
+          this.setState({ pendingInvites: this.props.club.inviteIDs });
+          this.fetchPendingInvites();          
+        }
     }
   }
 
+  fetchPendingInvites = async () => {
+
+    const invitedUsers = [];
+
+    await Promise.all(this.props.club.inviteIDs.map(async (inviteUserId) => {
```

Comment:
Refactor promise all to remove the async and await here

Reply:
Done.